vowpalwabbit/cb_adf.cc

```diff
@@ -367,7 +369,7 @@ void output_example(vw& all, cb_adf& c, example& ec, v_array<example*>* ec_seq)
 
   float loss = 0.;
 
-  uint32_t action = ec.pred.multiclass;
+  uint32_t action = ec.pred.a_s[0].action;
```

Comment:
It should, unless there are no actions for some bizarre reason.

Alekh

On Tue, Jun 7, 2016 at 10:27 AM, Luong Hoang <notifications@github.com>
wrote:

> In vowpalwabbit/cb_adf.cc
> <https://github.com/JohnLangford/vowpal_wabbit/pull/1006#discussion_r66080953>
> :
>
> > @@ -367,7 +369,7 @@ void output_example(vw& all, cb_adf& c, example& ec, v_array<example*>* ec_seq)
> >
> >    float loss = 0.;
> >
> > -  uint32_t action = ec.pred.multiclass;
> > +  uint32_t action = ec.pred.a_s[0].action;
>
> lacking a bit of context on pred.a_s, but is it always guaranteed to
> contain at least 1 item?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JohnLangford/vowpal_wabbit/pull/1006/files/73eb39a39cbabe685f34de771a5b2469bcda7f3c#r66080953>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAyIGaPAtrTWh10WeCUiEQRBGxeRzt9mks5qJX_hgaJpZM4Iu-Dw>
> .
>